### PR TITLE
Allow VIES Service timeout to be editable

### PR DESCRIPTION
**Pull request by chrisrollins65**

### CODE REVIEW

File: vat.go [hunk 1]
```diff
@@ -15,8 +15,17 @@ Get VAT rate that is currently in effect for a given country
...
 
+import "time"
+
 // ViesLookupService is the interface for the VIES VAT number validation service
 var ViesLookupService LookupServiceInterface = &viesService{}
 
 // UKVATLookupService is the interface for the UK VAT number validation service
 var UKVATLookupService LookupServiceInterface = &ukVATService{}
+
+var serviceTimeout = time.Duration(60) * time.Second
```
Comment (ripexz) [inline vat.go:26]: you can just do `var serviceTimeout =  time.Second * 60` 🤔

File: vat.go [hunk 2]
```diff
@@ -15,8 +15,17 @@ Get VAT rate that is currently in effect for a given country
...
+
+// SetServiceTimeout sets the timeout in seconds for the external VAT lookup services.
+func SetServiceTimeout(seconds int) {
+	serviceTimeout = time.Duration(seconds) * time.Second
```
Comment (ripexz) [inline vat.go:30]: arguably you could just accept `timeout time.Duration` and set `serviceTimeout =  timeout`  😅

Comment (chrisrollins65) [reply vat.go:30]: Thought I'd do it this way just to make it a bit easier for those implementing this, but I guess if I accept a Duration then they can set it to whatever they want and not necessarily seconds. Sure I'll change it

File: vat.go (hunk 1 above)
Comment (chrisrollins65) [reply vat.go:26]: You are correct. This was basically just copy-paste and I didn't really think about it

**Review by ripexz — APPROVED** [review]

**Pull request merged**